src/component/TimerArea/index.jsx

```diff
@@ -1,8 +1,20 @@
 import React from 'react';
+
+import Button from 'react-bootstrap/Button';
+import Card from 'react-bootstrap/Card';
 import Timer from '../Timer';
 
 const TimerArea = () => (
-  <Timer />
+  <Card>
+    <Card.Body className="text-center">
+      <h1>00:25:00</h1>
+      <Timer />
+      <div className="d-flex justify-content-around">
+        <Button variant="dark" size="lg">Start &#128525;</Button>
+        <Button variant="info" size="lg">Stop &#128564;</Button>
+      </div>
```

Comment:
This is no longer working because the `Timer` component has its own `button`s and time display. By doing what you're doing, you're displaying this:

![screenshot](https://user-images.githubusercontent.com/12049480/120423714-d03d1a80-c31f-11eb-97d2-028ee1c54e4a.png)

You need to merge the buttons inside the timer with the buttons you designed somehow. I'll leave that to you to figure out.